verify/verifier.go

```diff
@@ -124,16 +124,28 @@ func (v *Verifier) Commit(blob []byte) (*bn254.G1Affine, error) {
 // Verify regenerates a commitment from the blob and asserts equivalence
 // to the commitment in the certificate
 // TODO: Optimize implementation by opening a point on the commitment instead
-func (v *Verifier) VerifyCommitment(expectedCommit *common.G1Commitment, blob []byte) error {
+func (v *Verifier) VerifyCommitment(certCommitment *common.G1Commitment, blob []byte) error {
 	actualCommit, err := v.Commit(blob)
 	if err != nil {
 		return err
 	}
 
 	expectedX := &fp.Element{}
-	expectedX.Unmarshal(expectedCommit.X)
+	expectedX.Unmarshal(certCommitment.X)
```

Comment:
can we change all expected -> certCommitment

Reply:
[dc39a19](https://github.com/Layr-Labs/eigenda-proxy/pull/229/commits/dc39a19adc7105241a6240cb9b5a0ec61785b250)